Makefile

```diff
@@ -290,14 +290,7 @@ endif
 # libXSMM Backends
 ifneq ($(wildcard $(XSMM_DIR)/lib/libxsmm.*),)
   $(libceeds) : LDFLAGS += -L$(XSMM_DIR)/lib -Wl,-rpath,$(abspath $(XSMM_DIR)/lib)
-  $(libceeds) : LDLIBS += -lxsmm -ldl
-  MKL ?= 0
```

Comment:
Perhaps instead of (or in addition to) the `MKL` boolean, we should respond to `MKLROOT` being set.

Reply:
That would work. We need the BLAS/MKL fallback for performance reasons, so we can't simply use libxsmmnoblas.pc as discussed in the issue.

Reply:
Updated. Now we *should* detect MKLROOT and link against MKL if installed and BLAS otherwise.